back/app/controllers/web_api/v1/initiatives_controller.rb

```diff
@@ -41,11 +41,13 @@ def index_xlsx
     initiatives = SortByParamsService.new.sort_initiatives(initiatives, params, current_user)
 
     I18n.with_locale(current_user&.locale) do
-      xlsx = XlsxService.new.generate_initiatives_xlsx initiatives,
-        view_private_attributes: Pundit.policy!(current_user,
-          User).view_private_attributes?
-      send_data xlsx, type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
-        filename: 'initiatives.xlsx'
+      xlsx = XlsxService.new.generate_initiatives_xlsx(
+        initiatives,
+        view_private_attributes: policy(User).view_private_attributes?
```

Comment:
It is deliberate :) `policy` is using `Pundit::Context#policy!` internally. So, it will fail the same way if the policy cannot be found, but it saves us the need to specify the user explicitly. It uses the `pundit_user` instead (which includes the context we want to inject in policies).